CONTRIBUTING.rst

```diff
@@ -0,0 +1,217 @@
+
+  1. Go to the `SOLikeT GitHub Repository <https://github.com/simonsobs/soliket>`_
+  2. Click the **Fork** button (in the top-right-hand corner)
+  3. Choose where to create the fork, typically your personal GitHub account
+
+Next *clone* your fork. Cloning creates a local copy of the repository on your computer to work with. To clone your fork:
+
+::
+
+   git clone https://github.com/<your-account>/soliket.git
+
+
+Finally add the ``simonsobs`` repository as a *remote*. This will allow you to fetch changes made to the codebase. To add the ``simonsobs`` remote:
+
+::
+
+  cd soliket
+  git remote add simonsobs https://github.com/simonsobs/soliket.git
+
+
+Create a branch for your new feature
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a *branch* off the master branch. Working on unique branches for each new feature simplifies the development, review and merge processes by maintining logical separation. Branch names should have names prefixed with `dev-` and reflect the work going on within them. To create a feature branch:
+
+::
+
+  git checkout -b <your-branch-name> master
+
+
+**If you are not an SO GitHub org member**, create a *branch* off the ``simonsobs`` master branch. Working on unique branches for each new feature simplifies the development, review and merge processes by maintining logical separation. To create a feature branch:
+
+::
+
+  git fetch simonsobs
+  git checkout -b <your-branch-name> simonsobs/master
+
+
+Hack away!
+^^^^^^^^^^
+
+Write the new code you would like to contribute, remembering to abide by the `Code Style`_, and *commit* it to the feature branch on your local repository. Ideally commit small units of work often with clear and descriptive commit messages describing the changes you made. To commit changes to a file:
+
+::
+
+  git add file_containing_your_contribution
+  git commit -m 'Your clear and descriptive commit message'
+
+*Push* the contributions in your feature branch to your remote fork on GitHub:
+
+::
+
+  git push origin <your-branch-name>
+
+
+**Note:** The first time you *push* a feature branch you will probably need to use `--set-upstream origin` to link to your remote fork:
+
+::
+
+  git push --set-upstream origin <your-branch-name>
+
+
+Open a Pull Request
+^^^^^^^^^^^^^^^^^^^
+
+When you feel that work on your new feature is complete, you should create a *Pull Request*. This will propose your work to be merged into the main SOLikeT repository.
+
+  1. Go to `SOLikeT Pull Requests <https://github.com/simonsobs/soliket/pulls>`_
+  2. Click the green **New pull request** button
+  3. Click **compare across forks**
+  4. Confirm that the base fork is ``simonsobs/soliket`` and the base branch is ``master``
+  5. If you are making a contribution from a forl, confirm the head fork is ``<your-account>/soliket`` and the compare branch is ``<your-branch-name>``
+  6. Give your pull request a title and fill out the the template for the description
```

Comment:
Oh-- I didn't even know there was such thing as a "draft PR" (which I now see that this is...).  Maybe some suggestion regarding how to do that here?  (instead of my WIP suggestion)